.ci/metrics/requirements.lock.txt

```diff
@@ -186,7 +186,7 @@ charset-normalizer==3.4.0 \
     --hash=sha256:fe9f97feb71aa9896b81973a7bbada8c49501dc73e58a10fcef6663af95e5079 \
     --hash=sha256:ffc519621dce0c767e96b9c53f09c5d215578e10b02c285809f76509a3931482
     # via requests
-cryptography==43.0.3 \
+cryptography==44.0.1 \
```

Comment:
Bumping version without hash update is incorrect, we've tried that already in #17047 and it doesn't work.

Reply:
sigh, thanks